app/src/main/java/de/westnordost/streetcomplete/screens/main/map/LocationAwareMapFragment.kt

```diff
@@ -44,6 +45,15 @@ open class LocationAwareMapFragment : MapFragment() {
     /** The GPS trackpoints the user has walked */
     private var tracks: MutableList<ArrayList<Location>>
 
+    /** If we are actively recording track history */
+    var tracksRecording = false
+        private set
+
+    /** The GPS trackpoints the user has recorded */
+    private var _tracksRecorded: ArrayList<Trackpoint>
+
+    val tracksRecorded: List<Trackpoint> get() = _tracksRecorded
```

Comment:
I think it would be helpful to rename these variables to make differentiating between `Boolean` and `List<Trackpoint>` easier:

```diff
- tracksRecording
+ isRecordingTracks

- _tracksRecorded
+ _recordedTracks

- tracksRecorded
+ recordedTracks
```